swagger.yaml

```diff
@@ -1888,6 +1888,57 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/hosts/{host_id}/actions/reset-validation/{validation_id}:
+    patch:
+      tags:
+        - installer
+      summary: Reset failed validation.
+      operationId: ResetValidation
+      description: Reset failed validation.  The validation is executed during "preparing for installation" stage.
```

Comment:
```suggestion
      description: Reset failed host validation.  The validation is executed during "preparing for installation" stage.
```

Will it only work for validations executed during "preparing for installation" stage?

Reply:
That's the plan

Reply:
That's the plan

Reply:
Why limit it?

Reply:
I am not sure it makes sense for the others.  These validations run only once.  Others will run periodically anyway.

Reply:
Although I can remove the mention of "preparing for installation".